### ISSUE TITLE
FTP password generation

### ISSUE THREAD

**Issue by Barahten** [rclone/rclone#4387]
It would be nice to have a tool for generating passwords for FTP. It will be convenient for those who use the following construction
`rclone --config=<path_to_conf_file> ...`

**Comment by animosity22** [comment]
What are you trying to generate passwords for? The server FTP command?

**Comment by Barahten** [comment]
I wrote above what design we use to transfer files to FTP. The config file we use is not the one that the Rclone generates in the process of adding new storage. We created the config file and put it in a special folder on the web server and when starting Rclone we use the config path and then the standard commands.

For the test (tested on a Windows machine), we simply transferred the settings for connecting to the FTP from the generated config file to our file. But when transferring the project to the battle server, our parameters for connecting to the FTP will change. Then we can open our config file and enter new data to connect to the FTP, but we cannot enter the password. We know the unencrypted password for FTP, but as I understand it, Rсlone needs an encrypted password, because in the process it will try to decrypt it, otherwise it will not be able to connect to FTP
